app/assets/stylesheets/views/comments.scss

```diff
@@ -275,3 +275,59 @@
 .moderator-template-button:not(.active) {
   @extend .crayons-btn--outlined;
 }
+
+#toggle-comments-sort-dropdown {
+  cursor: pointer;
+}
+
+#comments-sort-dropdown-container {
+  z-index: var(--z-dropdown);
```

Comment:
(UI bug / blocking) - This applies an elevated z-index to the whole area, including the heading, when we only want it to apply to the actual dropdown. The result is that it's possible to scroll the heading into the top nav bar:

![heading present in top nav bar](https://user-images.githubusercontent.com/20773163/159685190-fb4cc0fa-b309-4cc6-83a9-2714d9bb2a4a.png)

You should be able to rely on the `crayons-dropdown` class on the dropdown content itself to control `z-index`, but let me know if you have any problems with that

Reply:
That was my initial thought as well. But it seemed to cause a UI issue. 
<img width="379" alt="Screenshot 2022-03-26 at 2 00 10 PM" src="https://user-images.githubusercontent.com/9512872/160231475-b9b5a37b-c59e-4948-a509-1399084e4eec.png">

Reply:
@aitchiss do let me know if you figure this one out